apertium-tur-uzb.tur-uzb.dix

```diff
@@ -4432,14 +4473,14 @@
        <e><p><l>ahşapçılık<s n="n"/></l><r>yogʻochsozlik<s n="n"/></r></p></e>
        <e><p><l>nişan<s n="n"/></l><r>orden<s n="n"/></r></p></e>
        <e><p><l>karın<s n="n"/></l><r>qorin<s n="n"/></r></p></e>
-       <e><p><l>üstün<s n="adj"/></l><r>ustun<s n="adj"/></r></p></e>
+       <e><p><l>sütün<s n="adj"/></l><r>ustun<s n="adj"/></r></p></e>
```

Comment:
Are you sure about this?

Reply:
the old one was true, "sütün= column" which is a noun.

Reply:
So,  "üstün" means "superior"(ustun in Uzbek), which is (adj),
and "sütün" means "column"(again, ustun in Uzbek), which is (noun).
Fixed it.